src/generic/grid.cpp

```diff
@@ -5889,6 +5886,69 @@ void wxGrid::RefreshBlock(int topRow, int leftCol,
     }
 }
 
+void wxGrid::RefreshArea(int areas)
+{
+    if ( areas == wxGA_All )
+    {
+        // Normally if we want to refresh the entire grid we call
+        // Refresh() instead, but to make this function consistent
+        // and correct, just forward to wxScrolledCanvas::Refresh()
+        // if the argument is wxGA_All.
+        wxScrolledCanvas::Refresh();
+        return;
+    }
+
+    // Corner area
+    if ( (areas & wxGA_Corner) != 0 )
+        m_cornerLabelWin->Refresh();
+
+    int cw, ch;
+    GetClientSize(&cw, &ch);
+
+    // Cells area
+    if ( (areas & wxGA_Cells) != 0 )
+    {
+        const wxRect rect = wxRect(GetRowLabelSize(), GetColLabelSize(), cw, ch);
+
+        wxScrolledCanvas::Refresh(true, &rect);
+    }
+
+    // RowLabels area
+    if ( (areas & wxGA_RowLabels) != 0 && GetRowLabelSize() > 0 )
+    {
+        const wxRect rect = wxRect(0, GetColLabelSize(), GetRowLabelSize(), ch);
+
+        wxScrolledCanvas::Refresh(true, &rect);
+    }
+
+    // ColLabels area
+    if ( (areas & wxGA_ColLabels) != 0 && GetColLabelSize() > 0 )
+    {
+        const wxRect rect = wxRect(GetRowLabelSize(), 0, cw, GetColLabelSize());
+
+        wxScrolledCanvas::Refresh(true, &rect);
+    }
+}
+
+void wxGrid::RefreshArea(wxGridArea area, const wxRect& rect)
+{
+    wxASSERT_MSG( !rect.IsEmpty(), "RefreshArea expects a valid rectangle to refresh" );
+
+    if ( area == wxGA_Corner )
```

Comment:
This function doesn't seem to be ever called with this area, is it?

Reply:
Removed now